packages/amplify-cli/src/extensions/amplify-helpers/resource-status.js

```diff
@@ -9,6 +9,7 @@ const _ = require('lodash');
 const { CLOUD_INITIALIZED, CLOUD_NOT_INITIALIZED, getCloudInitStatus } = require('./get-cloud-init-status');
 const { readJsonFile } = require('./read-json-file');
 const { ServiceName: FunctionServiceName, hashLayerResource } = require('amplify-category-function');
+const { CloudFormation } = require('aws-sdk');
```

Comment:
Use `aws-provider-cloudformation`

Reply:
It seems like CloudFormation from the `aws-provider-cloudformation` library can't be directly replaced without making other changes (getting a "CloudFormation is not a constructor" error).

Is there anything else I'd need to modify?

Reply:
remove this for now you cannot use a non configure aws-sdk directly